lib/pacemaker/pcmk_sched_utilization.c

```diff
@@ -287,32 +287,34 @@ sum_resource_utilization(pe_resource_t *orig_rsc, GList *rscs)
  * \internal
  * \brief Ban resource from nodes with insufficient utilization capacity
  *
- * \param[in]     rsc       Resource to check
- * \param[in,out] prefer    Resource's preferred node (might be updated)
+ * \param[in] rsc  Resource to check
+ *
+ * \return Allowed node for \p rsc with most spare capacity, if there are no
+ *         nodes with enough capacity for \p rsc and all its colocated resources
```

Comment:
I agree it's a little clunky (returning that the most capable node is `NULL` if any node has enough capacity; arguably returning a most capable node at all from `pcmk__ban_insufficient_capacity` just based on the name). That said, it's the best I can come up with too, it's private, and the doxygen block is clear about how it behaves. I'm good with this.